src/ray/core_worker/core_worker.cc

```diff
@@ -1198,6 +1199,17 @@ Status CoreWorker::SubmitActorTask(const ActorID &actor_id, const RayFunction &f
   return status;
 }
 
+Status CoreWorker::KillTask(const ObjectID &object_id, bool force_kill) {
```

Comment:
Should we rename this to `CancelTask` to match python API?

Reply:
Sure!